gradle.properties

```diff
@@ -91,8 +89,8 @@ graphqlToolVersion=0.14.0-20250523-094200-b95690f
 # Protoc Tool
 protocToolVersion=1.0.0-20250528-120700-b5350a0
 
-# OpenAPI Module
-openapiToolVersion=2.3.2
+# OpenAPI Tool
+openapiToolVersion=2.4.0
```

Comment:
We should use a timestamped version here right?

Reply:
Fixed by [d79ba88](https://github.com/ballerina-platform/ballerina-distribution/pull/6147/commits/d79ba889e775feff5cc71658eaf563de48675ec5)